client/my-sites/plugins/marketplace/marketplace-plugin-details/test/index.js

```diff
@@ -33,10 +34,14 @@ describe( '<PurchaseArea/> Plugin details next step tests', () => {
 				onAddYoastPremiumToCart={ onAddYoastPremiumToCart }
 				onNavigateToCheckout={ onNavigateToCheckoutMockFunction }
 				onNavigateToDomainsSelection={ onNavigateToDomainsSelectionMockFunction }
+				onRemoveEverythingFromCart={ onRemoveEverythingFromCart }
 			/>
 		);
 		const yoastPremiumButton = await marketplacePluginDetails.findByText( 'Buy Yoast Premium' );
 		await fireEvent.click( yoastPremiumButton );
+		//onAddPlugin call in the PurchaseArea component is async so we have to queue and await a promise  for the following mocks to be called
```

Comment:
Note that in general put one space after `//` like this: https://github.com/Automattic/wp-calypso/blob/trunk/client/types.ts#L58 . Also, there is an extra space between "promise" and "for"; let's remove it :)